website/content/docs/networking/cni.mdx

```diff
@@ -1,93 +1,62 @@
 ---
 layout: docs
-page_title: CNI
+page_title: CNI and bridge networking
 description: |-
-  Learn about Nomad's runtime environment variables, interpolation, caveats,
-  and more.
+  Learn about Nomad's support for Container Network Interface (CNI) plugins, Nomad's bridge configuration, how to configure CNI plugins, and how to use CNI networks with Nomad jobs. 
```

Comment:
This page reads more as a usage doc than the explanation it was before and removes a lot of that exposition - is that the intention?